Ravens Matrices.md

```diff
@@ -0,0 +1,36 @@
+# Summary
+> The series of tasks in this article worked to identify individual contributions to group performance to determine "team players." The Ravens Task in specific measured pattern recognition and spatial reasoning. This contributes to the study of IQ or liquid intelligence. 
+
+# References
+> Main paper: https://www.nber.org/system/files/working_papers/w27071/w27071.pdf
+> > Supporting paper: Raven, J. (2003). Raven’s Advanced Progressive Matrices, Test Books 1 and 2: Pearson Assessment.
+> > > CANNOT FIND ACCESS TO THIS PAPER
+
+# Stimuli
+## The visual components
+> Ravens matrices involve determining the missing element in a pattern which is generally presented in the form of a matrix.
+> In Raven’s Advanced Progressive Matrices, items are presented in black on a white background similar to the standard version, and items become increasingly difficult throughout the test. 
+> 
+![Example](images/Example.jpg)
+
+## Materials for alternative versions of the experiment 
+>  Raven’s Advanced Progressive Matrices was used for the task of the individual, but for the group task a very similar test called the Culture Fair Intelligence Test was used.
+> > In the group phase, all members gathered around a single laptop and collectively decided on an answer for each item.
+
+# Procedure
+## Steps
+> Step one: Have participants complete 14 Ravens items (even numbered items, ranging in difficulty from across sets I and II) in seven minutes 
+
+## Roles 
+> In the individual task all participants had the same role.
+> In the group task all individuals gathered around one laptop and collaborated, with one contributer also being assigned as the reporter.
+
+## Instructions
+> Participants are asked to look for a pattern and determine ‘what comes next’. Participants were asked to find the missing element (from options
+a to f). 
+
+# Criteria
+>  Points are given to each item where the next element in the pattern is determined correctly.
+
+## Performance calculation
+> No incentives were mentioned
```

Comment:
I think intelligence (IQ) was measured based on the correct number of elements in a pattern were determined